api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -511,7 +481,8 @@ private void createConfigFile(org.pmiops.workbench.firecloud.model.Workspace fcW
       Workspace workspace) {
     org.pmiops.workbench.db.model.Workspace dbWorkspace = workspaceService.getRequired(
         workspaceNamespace, workspaceId);
-    checkWorkspaceWriteAccess(workspaceNamespace, workspaceId);
+    workspaceService.enforceWorkspaceAccessLevel(workspaceNamespace,
```

Comment:
There are a bunch of methods in WorkspacesController which don't have this enforcement yet... did you mean to add this to them as well?

Reply:
Those are actually automatically handled by FC already. Anything which involves getWorkspace, or shareWorkspace, or anything there, handles its own permissions.